### ISSUE TITLE
legend location is reset with savefig

### ISSUE THREAD

**Issue by MarcMush** [jheinen/GR.jl#276]
savefig doesn't keep the location of the legend 
is this the correct way of indicating the legend location ?

`julia> legend("A", "B")`
`julia> plot(1:5, rand(5), 1:5, rand(5); location=4)`
`julia> savefig("out.png")`

with `plot()`:

![screenshot](https://user-images.githubusercontent.com/35898736/69876583-4b071e80-12c0-11ea-9896-775b28cae5a4.png)

output `out.png`:

![out](https://user-images.githubusercontent.com/35898736/69876582-4b071e80-12c0-11ea-9e55-07d91cd314ed.png)

**Comment by jheinen** [comment]
The treatment of figures in GR must be completely revised. I hope I'll finally get to it soon ...

**Comment by heliosdrm** [comment]
Maybe this was fixed by de79425 (released in GR 0.46.0)?

**Comment by MarcMush** [comment]
I guess it can be considered fixed, #287 is now a feature request then

my previous example is now working when rewritten like that:
```julia
julia> legend("A", "B")
julia> plot(1:5, rand(5), 1:5, rand(5); location=4)
julia> savefig("out.png"; location=4)
```
